components/ws-proxy/pkg/proxy/routes.go

```diff
@@ -153,16 +157,6 @@ func (ir *ideRoutes) HandleSSHHostKeyRoute(route *mux.Route, hostKeyList []ssh.S
 	})
 }
 
-func (ir *ideRoutes) HandleDirectIDERoute(route *mux.Route) {
```

Comment:
why do we remove it?

Reply:
This is no longer used, it was related to the old way of accessing webview resources if I'm not mistaken

Reply:
It is direct route to workspace for web sockets, extension resources and so on. If we remove it and someone get URL workspace they can access it.

Reply:
Isn't that handled by `HandleRoot` if not it wouldn't be working, it redirects to vscode server port for extension resources :thinking:

Reply:
you are right 👍